docs/src/test-advanced.md

```diff
@@ -42,7 +42,7 @@ These options would be typically different between local development and CI oper
 - `reportSlowTests: { max: number, threshold: number } | null` - Whether to report slow tests. When `null`, slow tests are not reported. Otherwise, tests that took more than `threshold` milliseconds are reported as slow, but no more than `max` number of them. Passing zero as `max` reports all slow tests that exceed the threshold.
 - `shard: { total: number, current: number } | null` - [Shard](./test-parallel.md#shards) information.
 - `updateSnapshots: boolean` - Whether to update expected snapshots with the actual results produced by the test run.
-- `webServer: { command: string, port?: number, cwd?: string, timeout?: number, env?: object }` - Launch a web server before the tests will start. It will automaticially detect the port when it got printed to the stdout.
+- `webServer: { command: string, port?: number, useExistingWebServer?: boolean, cwd?: string, timeout?: number, env?: object }` - Launch a web server before the tests will start. It will automaticially detect the port when it got printed to the stdout.
```

Comment:
Let's get ourselves a little wiggle room here:

```js
{
  launch: [
    {
      name: 'webserver',
      command: 'npx serve 5000',
      waitForPort: 5000,
      reuseExisting: !process.env.CI
    }
  ]
}

Reply:
Also this solution does not really help those interested in the server-per worker folks.